core/src/test/java/org/apache/iceberg/TestTables.java

```diff
@@ -80,21 +81,26 @@ public static Transaction beginCreate(File temp, String name, Schema schema,
   }
 
   public static Transaction beginReplace(File temp, String name, Schema schema, PartitionSpec spec) {
-    return beginReplace(temp, name, schema, spec, SortOrder.unsorted(), ImmutableMap.of());
+    return beginReplace(temp, name, schema, spec, SortOrder.unsorted(), ImmutableMap.of(), null);
   }
 
   public static Transaction beginReplace(File temp, String name, Schema schema, PartitionSpec spec,
-                                         SortOrder sortOrder, Map<String, String> properties) {
-    TestTableOperations ops = new TestTableOperations(name, temp);
-    TableMetadata current = ops.current();
+      SortOrder sortOrder, Map<String, String> properties) {
```

Comment:
Tiny nit: We are a bit inconsistent with this within Iceberg code but we are supposed to align continuing args with the first arg of the method.
```java
public static foo (int x, int y,
                   int z)
```

Reply:
would be nice if we have a template to enforce the format

Reply:
https://iceberg.apache.org/#community/#setting-up-ide-and-code-style - Exists, I'm not sure if your checkstyle files include this particular rule though. We should probably add it though

Reply:
I recently looked into checkstyle and found that it allows more spaces than are required. This is why we get extra indentation in a lot of places. I'd prefer to turn that off, but we'd actually have to make a ton of changes. I'm debating whether it's worth it.

Reply:
I'd rather we just have one way that passes checkstyle, although I know that patch will be a mess for folks with upstream branches and I personally wouldn't want to deal with it